server/channels/store/sqlstore/status_store.go

```diff
@@ -17,10 +17,28 @@ import (
 
 type SqlStatusStore struct {
 	*SqlStore
+
+	statusSelectQuery sq.SelectBuilder
 }
 
 func newSqlStatusStore(sqlStore *SqlStore) store.StatusStore {
-	return &SqlStatusStore{sqlStore}
+	s := SqlStatusStore{
+		SqlStore: sqlStore,
+	}
+
+	manualColumnName := quoteColumnName(s.DriverName(), "Manual")
+	s.statusSelectQuery = s.getQueryBuilder().
+		Select(
+			"COALESCE(UserId, '') AS UserId",
+			"COALESCE(Status, '') AS Status",
+			fmt.Sprintf("COALESCE(%s, FALSE) AS %s", manualColumnName, manualColumnName),
+			"COALESCE(LastActivityAt, 0) AS LastActivityAt",
+			"COALESCE(DNDEndTime, 0) AS DNDEndTime",
+			"COALESCE(PrevStatus, '') AS PrevStatus",
```

Comment:
@agnivade, would love your thoughts here. In "general" we don't `COALESCE` everywhere even though most of our code would fail if a customer inserted `NULL` columns. Is this a pattern we should adopt more holistically going forward?

Reply:
No, my recommendation has been to add `NOT NULL` columns for new tables going forward. We have been very lenient in our DB strictness which has caused various issues such as these.

Reply:
Fully aligned on new tables being `NOT NULL`. Should we adopt this pattern for existing tables, or plan a migration that solves this going forward? (Mostly wanting to avoid playing whack-a-mole as this issue crops up from time to time.)

Reply:
Doing a migration will be difficult given our non-blocking requirements. I'd be happy with a middle ground where we adopt this pattern for the most commonly used tables, and leave the rest with the whack-a-mole approach.